ListingService/Sources/SubredditRoute.swift

```diff
@@ -0,0 +1,19 @@
+import CoreAPI
+import Foundation
+
+public enum SubredditRoute: APIRoute {
+    case popular
+    case news
+    case other(String)
+
+    public var path: String {
+        switch self {
+        case .popular:
+            return "/r/popular"
+        case .news:
+            return "/r/news"
+        case .other(let name):
+            return "/r/" + name
```

Comment:
This works well for subreddits (as it's the subreddit route), but there is no route for just reddit.com (no route). When we get a login working, the default "/" route might be a good option to have available.

If we wanted to filter on hot/new/rising, the url would be "/new" "/rising" etc (which should be handled by another route imho)

Reply:
BTW, for your reference, review is required by reviewers with write access